doc/source/examples/ex_python_uv.rst

```diff
@@ -0,0 +1,167 @@
+.. _example_python_uv:
+
+.. note::
+
+    Go to the `bottom of this page`_ to download the ZIP file for the uv example.
+
+Running arbitrary Python scripts on HPS
+=======================================
+
+This example shows how arbitrary Python scripts can be run on HPS, by
+using `uv <https://docs.astral.sh/uv/>`__ to generate the required
+environments on the fly.
+
+The example sets up a project that shall plot ``sin(x)`` using numpy and
+matplotlib, and then save the figure to a file.
+
+The main feature enabling uv to take care of the environment setup is
+the metadata header present in the ``eval.py`` script, which defines the
+dependencies:
+
+.. code:: python
+
+   # /// script
+   # requires-python = "==3.12"
+   # dependencies = [
+   #     "numpy",
+   #     "matplotlib"
+   # ]
+   # ///
+
+More information can be found `here
+(python.org) <https://packaging.python.org/en/latest/specifications/inline-script-metadata/#inline-script-metadata>`__
+and `here
+(astral.sh) <https://docs.astral.sh/uv/guides/scripts/#running-a-script-with-dependencies>`__.
+
+Prerequisites
+=============
+
+In order for the example to run, ``uv`` must be installed and registered
+on the scaler/evaluator. Installation instructions can be found
+`here <https://docs.astral.sh/uv/getting-started/installation/>`__.
+
+Once uv is installed, the application must be registered in the
+scaler/evaluator with the following properties:
+
+.. list-table::
+   :header-rows: 1
+
+   * - Property
+     - Value
+   * - Name
+     - uv
+   * - Version
+     - 0.7.19
+   * - Installation Path
+     - /path/to/uv
+   * - Executable
+     - /path/to/uv/bin/uv
+
+Note that the version should be adjusted to the case at hand.
```

Comment:
This sentence is rather formal and ambiguous. Consider rewriting:
Be sure to adjust the version to the one you have installed.